tests/framework-tests/preferences/js/PrefsEditorTestsCommon.js

```diff
@@ -39,3 +39,20 @@ fluid.tests.mergeMembers = function (memberDef) {
     }
     return togo;
 };
+
+// An add-on grade to override distributeOptions in fluid.prefs.separatedPanel component whose targets are
+// outside of the separatedPanel component
+fluid.defaults("fluid.tests.prefs.overrideUnOutreach", {
```

Comment:
Slightly odd name - perhaps something like overrideBuilderDependencies or something which mentions the name of which component is not present in the test configuration?